src/Public/Security.ps1

```diff
@@ -123,7 +123,7 @@ function Add-PodeSecurityHeader {
         [string]
         $Name,
 
-        [Parameter()]
+        [Parameter(Mandatory = $true)]
```

Comment:
The `Mandatory = $true` needs removing, as this will break a lot of places that call `Add-PodeSecurityHeader`. Given this is also a public function, it's also a breaking change.

The removed `if (![string]::IsNullOrWhiteSpace($Value)) {` needs to be re-introduced. Or looking at it, a simpler check at the top of the function:

```powershell
if ([string]::IsNullOrWhiteSpace($Value)) {
    return
}
```